wire-ios-request-strategy/Sources/Request Strategies/Conversation/ConversationRequestStrategy.swift

```diff
@@ -845,6 +845,7 @@ class ConversationByQualifiedIDListTranscoder: IdentifierObjectSyncTranscoder {
 
         for qualifiedID in conversations {
             let conversation = ZMConversation.fetchOrCreate(with: qualifiedID.uuid, domain: qualifiedID.domain, in: context)
+            conversation.hasIncompleteMetadata = true
```

Comment:
When does this method get called? I'm wondering if we can avoid the Core Data attribute (to try minimize state) to store `hasIncompleteMetadata` and instead make it computed.